tools/capsule_benchmark/README.md

```diff
@@ -0,0 +1,40 @@
+# Capsule Benchmark
+This tool is used to produce benchmark result graphs for one or more capsules. 
+
+The tool will benchmark the capsule at different levels of parallelism. 
+Parallelism is defined as how many concurrent requests are being sent to the
+capsule at any given time. For example, a parallelism of 3 is equivalent to 
+3 threads pushing requests to the capsule as fast as they can be completed. 
+
+The reason that parallelism is a focus on this benchmark is because many deep 
+learning algorithms benefit from 'batching' requests. That is, sending multiple
+images or inputs to the GPU / other devices concurrently. 
+
+## Usage
+
+### Install Dependencies
+First, install the `vcap` and `vcap_utils` libraries. Then run:
+```
+pip3 install -r requirements.txt
+```
+
+### Running the script
+
+Store one or more unpackaged capsules in a directory, in this example called
+capsules`. Then run the script and point it to the directory with the
```

Comment:
I think there's a missing grave accent boy here. A grave mistake.

Reply:
a grave mistake indeed. I have corrected it!